go.mod

```diff
@@ -44,6 +44,7 @@ require (
 )
 
 replace (
+	github.com/codeready-toolchain/api => github.com/sbryzak/api v0.0.0-20210511043252-f5014d872796
```

Comment:
Just a reminder.

Reply:
Shouldn't be this done later? I thought that this PR should verify the migration and also that UserSignup controller correctly uses the approved state (it's at least paired with that one). Removal should be done later in another PR when everything is migrated.
In addition, before removing it we should also update the registration service as well as sandbox-sre, shouldn't we?

Reply:
I've got the registration service PR ready to go, but I won't pull the trigger on that until this one is merge otherwise the e2e tests CI job will complain about too many branches with the same name.  Will discuss with @alexeykazakov tomorrow the best way to proceed with this set of changes for minimal disruption.

Reply:
Yeah I guess it's OK to update the api dependency in e2e here if we don't need it in the code anymore. But what about keeping it for testing the migration: https://github.com/codeready-toolchain/toolchain-e2e/pull/298#discussion_r630726134 ?

Reply:
We are now testing the migration, so the api dependency cannot be updated as of yet.